package.json

```
@@ -88,11 +89,9 @@
     ]
   },
   "pnpm": {
-    "overrides": {
-      "typescript@<5.0.4": "^5.0.4"
-    },
     "onlyBuiltDependencies": [
       "esbuild"
     ]
-  }
+  },
+  "packageManager": "pnpm@10.13.1"
```

```suggestion
  }
```

Seems this is causing a conflict since the workflow uses latest. Was this forcefully added by pnpm?